MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -467,4 +467,9 @@
 - (void)getDeviceInformationWithParameters:(nullable MSALParameters *)parameters
                            completionBlock:(nonnull MSALDeviceInformationCompletionBlock)completionBlock;
 
+/**
+   A boolean indicates if a compatible broker is present in device.
+*/
+- (BOOL)isCompatibleBrokerAvailable;
```

Comment:
nit: should this be a readonly property?
nit2: should we indicate that this only applies to AAD in the method name?

Reply:
Sure, let me address it :)

Reply:
addressed.